### ISSUE TITLE
Tin for days

### ISSUE THREAD

**Issue by Ethryan** [GTNewHorizons/GT-New-Horizons-Modpack#11251]
### Your GTNH Discord Username

Ethryan

### Your Pack Version

2.2.0.0-Pre1

### Your Server

Private server

### Type of Server

Thermos

### Your Expectation

Not getting 12 stacks of tin from 1 stack of ore.

### The Reality

![unknown](https://user-images.githubusercontent.com/3237986/189373610-f3360672-2e25-45aa-8fd1-1a61a40437c3.png)

### Your Proposal

Maybe smelt into nuggets instead of complete ingots,

1 Ground Cassiterite Sand to 3 Tin Nuggets.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by NickBjurstrom** [comment]
Was the goal to change cass sand/ore (it's both) to nuggets when smelted directly? I'd say just revert back to what we used to have cass = 2 ingots smelted directly, or macerated into 4x crushed.

**Comment by chochem** [comment]
that was done after the electrolysis got changed. not a bug. Iirc you should now get the same amount of oxygen and tin from electrolysis as you did before. Just furnace is better.

**Comment by Hikari1nVoid** [comment]
GTNH-style chemistry applies here lol.

**Comment by Shadowsoftime99** [comment]
I'm probably just stating the obvious, but the cleanest way to change the smelt so that it aligns with old versions would be to make each dust smelt into 3 nuggets. One third of an ingot, which is basically what the dust electrolyzes into as well.

Before the chemistry changes it was 4 crushed per ore, 1 Tin 2 Oxygen per electrolyzed dust (1 Tin only if smelted). So, 12 crushed per orem and 1/3 Tin 2/3 Oxygen per crushed would fit.

**Comment by chochem** [comment]
yea nuggets were talked about at the time but nobody actually liked the idea. Same reason we are getting rid of tiny and small dusts more and more, this small parts are super annoying. Additionally you dont even have the steam alloy smelter yet.

**Comment by leagris** [comment]
It was changed in [fix small ore and cass ore GT5-Unofficial#1150 ](https://github.com/GTNewHorizons/GT5-Unofficial/pull/1150/files/31f3f092f2503e7d99abc52a027b3af1f411e2dd#diff-faf76538da89119ddb2b992386b619fd4152e300ddfaf76538da89119ddb2b992386b619fd4152e305cf01989b79d036d13690fR1272-R1273)
```diff
--- a/src/main/java/gregtech/api/enums/Materials.java
+++ b/src/main/java/gregtech/api/enums/Materials.java
@@ -1569,2 +1569,2 @@ public class Materials implements IColorModulationContainer, ISubTagContainer {
-        Cassiterite.setOreMultiplier(2).setSmeltingMultiplier(2);
-        CassiteriteSand.setOreMultiplier(2).setSmeltingMultiplier(2);
+        Cassiterite.setOreMultiplier(6).setSmeltingMultiplier(2);
+        CassiteriteSand.setOreMultiplier(6).setSmeltingMultiplier(2);
```
